personalize.admin.campaign.inc

```diff
@@ -496,14 +489,11 @@ function personalize_campaign_wizard_variations($form, &$form_state, $agent_data
     // Header information.
     $edit_link = module_invoke($option_set_plugin['module'], 'personalize_edit_link', $option_set);
     $delete_link = module_invoke($option_set_plugin['module'], 'personalize_delete_link', $option_set);
-    $has_report = $agent_instance instanceof PersonalizeAgentReportInterface && !empty($agent_data->started);
     $section['option_set_' . $option_set->osid]['summary'] = array(
       '#type' => 'markup',
       '#markup' => theme('personalize_wizard_variations_header', array(
         'variation_title' => $option_sets_labels[$option_set->osid],
         'variation_count' => count($option_set->options),
-        'report' => $has_report ? $agent_instance->renderStatsForOptionSet($option_set, $agent_data->started) : array(),
```

Comment:
Should we even keep renderStatsForOptionSet?  It only existed to be able to show some quick in-line numbers here in the option set display.